packages/ActionMenu/src/ActionMenu.tsx

```diff
@@ -0,0 +1,253 @@
+    if (currentFocusedOption) {
+      currentFocusedIndex = enabledOptions.findIndex(
+        (enabledOption) => enabledOption.value === currentFocusedOption.value
+      );
+    }
+
+    switch (direction) {
+      case 'up':
+        setCurrentFocusedOption(
+          enabledOptions[
+            currentFocusedIndex > 0
+              ? currentFocusedIndex - 1
+              : enabledOptions.length - 1
+          ]
+        );
+        break;
+      case 'down':
+        setCurrentFocusedOption(
+          enabledOptions[(currentFocusedIndex + 1) % enabledOptions.length]
+        );
+        break;
+      case 'last':
+        setCurrentFocusedOption(enabledOptions[enabledOptions.length - 1]);
+        break;
+      default:
+        setCurrentFocusedOption(enabledOptions[0]);
+        break;
+    }
+  };
+
+  const handleOnKeyDown = (
+    keyboardEvent: React.KeyboardEvent<HTMLDivElement>
+  ): void => {
+    switch (keyboardEvent.key) {
+      case Keys.Escape:
+        if (showMenu) {
+          toggleMenu(showMenu);
+        }
+        break;
+      case Keys.Enter:
+        keyboardEvent.preventDefault();
+        keyboardEvent.stopPropagation();
+        if (currentFocusedOption) {
+          selectOption(currentFocusedOption);
+        }
+        if ((!currentFocusedOption && showMenu) || !showMenu) {
+          toggleMenu(!showMenu);
+        }
+        break;
+      case Keys.Space:
+        if (!showMenu) {
+          toggleMenu(true);
+        }
+        break;
+      case Keys.ArrowUp:
+        keyboardEvent.preventDefault();
+        keyboardEvent.stopPropagation();
+
+        focusOption('up');
+        break;
+      case Keys.ArrowDown:
+        keyboardEvent.preventDefault();
+        keyboardEvent.stopPropagation();
+
+        focusOption('down');
+        break;
+      case Keys.Home:
+        keyboardEvent.preventDefault();
+        keyboardEvent.stopPropagation();
+
+        focusOption('first');
+        break;
+      case Keys.End:
+        keyboardEvent.preventDefault();
+        keyboardEvent.stopPropagation();
+
+        focusOption('last');
+        break;
+      case Keys.Tab:
+        if (showMenu) {
+          toggleMenu(showMenu);
+        }
+
+        break;
+      default:
+        break;
+    }
+  };
+
+  const actionMenuClassname = cx('ids-action-menu', className);
+
+  return (
+    <div className={actionMenuClassname} data-test={dataTest} {...rest}>
+      <Dropdown
+        key="selectDropdown"
+        content={
+          <List
+            options={actionMenuOptions}
+            onOptionFocus={hoverOption}
+            onOptionChange={selectOption}
+            focusedOption={currentFocusedOption}
+          />
+        }
+        isOpen={showMenu}
+        className="ids-action-menu__dropdown"
+        position={position}
+        onClose={() => toggleMenu(false)}
+        renderReference={({ ref, getReferenceProps }: ReferenceProps) => {
+          return (
+            <IconButton
```

Comment:
You have to let the user pass what he wants, it can be a button or something else. You can use the props children to do this